Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -208,10 +216,12 @@ public struct PackageBuilder {
     }
 
     /// Returns path to all the items in a directory.
+    /// FIXME: This is generic functionality, and should move to FileSystem.
```

Comment:
Thanks for the link to #521.  But it seems orthogonal to me, because part of the point of `FileSystem` is that the client code (`PackageBuilder` in this case) shouldn't need to know or care whether it's given a file system that maps directly to the underlying file system, to a git repo or `cpio` file or whatever, or to a `RerootedFileSystemView` on top of one of those other kinds of file system.

In all cases the API deals in terms of just paths (currently called `AbsolutePath` but eventually intending to be renamed to `Path`).  And it doesn't change the fact that sometimes you want to walk a file system hierarchy.  Other times you just want to get the names of the entries in a single directory.